### ISSUE TITLE
TOTALVI.get_normalized_expression omits learned protein capture efficiency

### ISSUE THREAD

**Issue by arberqoku** [scverse/scvi-tools#4022]
### Problem

`TOTALVI.get_normalized_expression()` appears to omit the learned per-protein, per-batch capture-efficiency term from its protein readout. There is no separate public `get_normalized_protein_intensity()` method.

The public method computes protein values from unscaled `py_` rates:

https://github.com/scverse/scvi-tools/blob/1.5.1/src/scvi/model/_totalvi.py#L544-L550

### Minimal reproduction

Reproduced with `scvi-tools==1.5.1` in a fresh Python 3.12.7 environment.

```python
import anndata as ad
import numpy as np
import scvi
import torch

adata = ad.AnnData(np.ones((8, 3), dtype=np.float32))
adata.obs["batch"] = "batch_0"
adata.obsm["protein_expression"] = np.ones((8, 2), dtype=np.float32)

scvi.model.TOTALVI.setup_anndata(
    adata,
    batch_key="batch",
    protein_expression_obsm_key="protein_expression",
)

model = scvi.model.TOTALVI(adata, n_latent=1)

# set a non-unit efficiency deterministically to isolate the readout scale.
with torch.no_grad():
    model.module.log_per_batch_efficiency.fill_(np.log(2.0))

model.module.eval()

torch.manual_seed(123)
_, public_protein = model.get_normalized_expression(
    adata=adata,
    include_protein_background=True,
    sample_protein_mixing=False,
    n_samples=1,
    return_mean=True,
    return_numpy=True,
    silent=True,
)

# reconstruct the same forward pass and compare unscaled and scaled rates.
torch.manual_seed(123)
tensors = next(iter(model._make_data_loader(adata=adata, batch_size=8)))

with torch.inference_mode():
    _, generative = model.module.forward(
        tensors,
        inference_kwargs={"n_samples": 1},
        generative_kwargs={},
        compute_loss=False,
    )

pi = torch.sigmoid(generative["py_"]["mixing"])

unscaled = (
    generative["py_"]["rate_fore"] * (1 - pi)
    + generative["py_"]["rate_back"] * pi
).cpu().numpy()

efficiency_scaled = (
    generative["py_norm_"]["rate_fore"] * (1 - pi)
    + generative["py_norm_"]["rate_back"] * pi
).cpu().numpy()

print(f"scvi-tools={scvi.__version__}")
print(f"public-vs-unscaled={np.max(np.abs(public_protein - unscaled)):.3e}")
print(f"public/scaled={np.mean(public_protein / efficiency_scaled):.6f}")
print(
    f"1/per_batch_efficiency={np.mean(1.0 / generative['per_batch_efficiency'].cpu().numpy()):.6f}"
)
```

Output:

```text
scvi-tools=1.5.1
public-vs-unscaled=0.000e+00
public/scaled=0.500000
1/per_batch_efficiency=0.500000
```

The public protein readout is exactly the unscaled mixture and is inflated by `1 / per_batch_efficiency` relative to the efficiency-scaled mixture.

### Expected behavior

The protein output should be on the same scale as the protein reconstruction likelihood.

The likelihood uses efficiency-scaled rates:

```python
mu1 = per_batch_efficiency * py_["rate_back"]
mu2 = per_batch_efficiency * py_["rate_fore"]
```

https://github.com/scverse/scvi-tools/blob/1.5.1/src/scvi/module/_totalvae.py#L329-L342

`generative()` already constructs these scaled rates as `py_norm_`:

https://github.com/scverse/scvi-tools/blob/1.5.1/src/scvi/module/_totalvae.py#L474-L483

Could you confirm whether `get_normalized_expression()` should use the efficiency-scaled protein rates?

### Versions

```python
| Package      | Version               |
| ------------ | --------------------- |
| anndata      | 0.13.3.post0          |
| lightning    | 2.6.6                 |
| numpy        | 2.5.3                 |
| pandas       | 3.0.5                 |
| scanpy       | 1.12.4                |
| scvi-tools   | 1.5.1                 |
| scipy        | 1.18.1                |
| torch        | 2.14.0 (2.14.0+cu130) |
| torchmetrics | 1.9.0                 |

| Dependency          | Version     |
| ------------------- | ----------- |
| matplotlib          | 3.11.2      |
| donfig              | 0.8.1.post1 |
| scikit-learn        | 1.9.1       |
| mpmath              | 1.3.0       |
| tqdm                | 4.70.1      |
| h5py                | 3.16.0      |
| google-crc32c       | 1.8.0       |
| setuptools          | 78.1.0      |
| docrep              | 0.3.2       |
| legacy-api-wrap     | 1.5         |
| xarray              | 2026.7.0    |
| fast-array-utils    | 1.5         |
| ml-collections      | 1.1.0       |
| typing-inspection   | 0.4.4       |
| pydantic_core       | 2.46.5      |
| natsort             | 8.4.0       |
| packaging           | 26.3        |
| pillow              | 12.3.0      |
| pydantic            | 2.13.5      |
| cloudpickle         | 3.1.2       |
| cuda-bindings       | 13.4.1      |
| opt_einsum          | 3.4.0       |
| cycler              | 0.12.1      |
| zarr                | 3.4.0       |
| python-dateutil     | 2.9.0.post0 |
| absl-py             | 2.5.0       |
| joblib              | 1.6.0       |
| numba               | 0.67.0      |
| six                 | 1.17.0      |
| attrs               | 26.1.0      |
| annotated-types     | 0.8.0       |
| rich                | 15.0.0       |
| mudata              | 0.4.1       |
| fonttools           | 4.65.0      |
| fsspec              | 2026.7.0    |
| narwhals            | 2.26.0      |
| session-info2       | 0.4.2       |
| pyparsing           | 3.3.2       |
| python-dotenv       | 1.2.3       |
| PyYAML              | 6.0.3       |
| scverse-misc        | 0.1.6       |
| lightning-utilities | 0.15.3      |
| threadpoolctl       | 3.7.0       |
| numcodecs           | 0.16.5      |
| triton              | 3.8.0       |
| kiwisolver          | 1.5.1       |
| typing_extensions   | 4.16.0      |
| sparse              | 0.19.2      |
| msgspec             | 0.21.1      |
| pyro-ppl            | 1.9.1       |
| statsmodels         | 0.15.0      |
| cuda-pathfinder     | 1.8.1       |
| pydantic-settings   | 2.15.0      |
| llvmlite            | 0.49.0      |
| sympy               | 1.14.0      |

| Component | Info                                                                          |
| --------- | ----------------------------------------------------------------------------- |
| Python    | 3.12.7 | packaged by conda-forge | (main, Oct  4 2024, 16:05:46) [GCC 13.3.0] |
| OS        | Linux-6.8.0-124-generic-x86_64-with-glibc2.39                                 |
| Updated   | 2026-09-16 18:42                                                              |
```

**Comment by ori-kron-wis** [comment]
Thanks @arberqoku , I added a flag to incorporate this normalization in get_normalized_expression.
The fix is in this PR (not the automatic bot one from before): https://github.com/scverse/scvi-tools/pull/4033
